test/test_sparse.py

```diff
@@ -2256,6 +2256,8 @@ def test_log1p(self, device, dtype, coalesced):
                 device=device,
                 dtype=dtype
             )
+            # empty tensors are coalesced at creation (nnz < 2) we must force the uncoalesced state
+            input_uncoalesced._coalesced_(False)
```

Comment:
This change preserves the test case, we must now set coalesced `False` manually for empty input.